### PR TITLE
Server setup

### PR DESCRIPTION
**Pull request by Newbology**

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    store: new redis({ url: `${REDIS_URL}:${REDIS}`, logErrors: true }),
+    secret: SESSION_SECRET,
+    resave: false,
+    saveUninitialized: false,
+    cookie: { secure: ENV === 'production' }
+  })
+);
+
+app.use(passport.initialize());
+app.use('/rolodex', rolodex);
```
Comment (taesup) [inline server/server.js]: You probably want to name the route according to the data model that you'll be sending back. Either Users or Contacts. Your data models are defined by your tables in most cases.

**Review by taesup — CHANGES_REQUESTED** [review]

**Comment by Newbology** [comment]
meant to branch off for migrations, committed and pushed code after making PR

File: database/migrations/20190219185900_foreignkey.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.table('users', table => {
+    table.integer('contacts_id').references('contacts_id').inTable('contacts').notNullable();
+  });
+};
+
+exports.down = function(knex, Promise) {
+  return knex.schema.dropTable('users');
```
Comment (taesup) [inline database/migrations/20190219185900_foreignkey.js:8]: For this migration you shouldn't be dropping the table, that's not the opposite of what the up function is doing. It should be dropping that column you just created. Nothing you can do now for this though but keep note of what you should be doing in a down function.

File: server/server.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+const PORT = process.env.EXPRESS_CONTAINER_PORT;
+const ENV = process.env.NODE_ENV || 'development';
+const SESSION_SECRET = process.env.SESSION_SECRET || 'keyboard cat';
+const REDIS = process.env.REDIS_HOST_PORT;
+const REDIS_URL = process.env.REDIS_URL;
+const app = express();
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+app.use(express.static('public'));
```
Comment (taesup) [inline server/server.js]: express.static should come before bodyParser

**Review by taesup — CHANGES_REQUESTED** [review]

**Review by taesup — APPROVED** [review]

**Pull request merged**